backend/api.py

```diff
@@ -197,27 +197,6 @@ def post_edited_niceties():
         return jsonify({'authorized': "false"})
 
 
-@app.route('/api/v1/admin-edit-niceties', methods=['POST'])
-@needs_authorization
-def get_niceties_to_edit():
```

Comment:
Oh wow, this is the thing you mentioned last week - these method names are completely backwards! Glad to get this gone.